inference-engine/tests/functional/shared_test_classes/src/base/layer_test_utils.cpp

```diff
@@ -328,6 +336,23 @@ void LayerTestsCommon::ConfigureNetwork() {
             out.second->setPrecision(outPrc);
         }
     }
+
+    if (!inputDynamicShape.empty()) {
```

Comment:
MINOR!!! such code contains reshape and other could be separated in other function. Because it look like as a separated block..
```
if (!inputDynamicShape.empty()) {
     SetShapeForCNNNet(...)
}
```

Moreover, there are a lot of `if` constructions. I don't like such approach

Reply:
Actually separating block needs to be considered, but to support eltwise style input, many if were added. So if it is not supported, the codes can be simplified even a little.

Reply:
Do you mean 'multiinput' using 'eltwise style input'?

Reply:
Actually two thing are confused... The `multiple target shapes` means one or more target static shapes under single input dynamic shape, and the `eltwise style input` means two inputs (or more) those are used in kind of eltwise.